docs/api/datasets.rst

```diff
@@ -329,6 +329,11 @@ VHR-10
 
 .. autoclass:: VHR10
 
+WesternUSALiveFuelMoisture
```

Comment:
I would separate this out into separate words. Will also need to change in the table. This will help split the title onto multiple lines in the table.